src/Workspaces/Core/Portable/GeneratedCodeRecognition/GeneratedCodeRecognitionServiceFactory.cs

```diff
@@ -37,12 +37,10 @@ private static bool IsFileNameForGeneratedCode(string fileName)
                 {
                     fileName = Path.GetFileNameWithoutExtension(fileName);
 
-                    if (fileName.EndsWith("AssemblyInfo", StringComparison.OrdinalIgnoreCase) ||
-                        fileName.EndsWith(".designer", StringComparison.OrdinalIgnoreCase) ||
+                    if (fileName.EndsWith(".designer", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
isn't this generic service? is that okay to change default to meet suppression's need? how about other consumers?